app/views/service_providers/_form.html.erb

```diff
@@ -52,18 +52,18 @@
     <%= form.label :identity_protocol,
                    label: 'Authentication protocol',
                    class: 'usa-label' %>
-    <%= content_tag(:ul, class: 'usa-input-list') do
-      ServiceProvider.identity_protocols.each_key do |protocol| %>
-        <%= content_tag(:li) do %>
+    <div>
```

Comment:
Is the wrapping `<div>` necessary?

Reply:
The [changelog](https://github.com/uswds/uswds/releases/tag/v2.0.0-beta.7) seemed to indicate replacing the `<ul>` with a `<div>` but i may have read it incorrectly.

Reply:
updated in https://github.com/18F/identity-dashboard/pull/740/commits/20e94541458f876cf2ab980a6a9c511e0bce11dc